### PR TITLE
Start tauri-based frontend from main binary

### LINKED ISSUE #96: Run frontend in application
I don't care if its [gluon](https://github.com/gluon-framework/gluon), [tauri](https://tauri.app/) or electron or whatever, but we should have _some_ way of actually launching bazed as an application rather than just as a website. A prototype is more than enough here!

### PR DESCRIPTION
**Pull request by elkowar**
This PR finally integrates tauri, and thus fixes #96.

### CODE REVIEW

File: crates/bazed-tauri/Cargo.toml [hunk 1]
```diff
@@ -1,14 +1,21 @@
 [package]
 name = "bazed-tauri"
 version = "0.0.0"
-description = "A Tauri App"
```
Comment (buffet) [inline crates/bazed-tauri/Cargo.toml]: This shouldn't be part of this commit.

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -15,12 +15,13 @@ jobs:
...
     runs-on: ubuntu-latest
-
     steps:
+    - name: Install dependencies
+      run: sudo apt-get update && sudo apt-get install libgtk-3-dev libsoup2.4-dev libjavascriptcoregtk-4.0-dev libwebkit2gtk-4.0-dev
     - uses: actions/checkout@v3
     - name: Setup rust toolchain
       run: rustup show
-    - uses: EmbarkStudios/cargo-deny-action@v1
+    #- uses: EmbarkStudios/cargo-deny-action@v1
```
Comment (buffet) [inline .github/workflows/ci.yaml]: This needs to be uncommented again.

**Review by buffet — CHANGES_REQUESTED** [review]
webkit2gtk-4.0 should really be libwebkit2gtk-4.0 for consistency

**Review by buffet — APPROVED** [review]

**Comment by gwimm** [comment]
```
   Compiling bazed-tauri v0.0.0 (C:\Users\Cpli\Documents\proto\bazed\crates\bazed-tauri)
error: failed to run custom build command for `bazed-tauri v0.0.0 (C:\Users\Cpli\Documents\proto\bazed\crates\bazed-tauri)`

Caused by:
  process didn't exit successfully: `C:\Users\Cpli\Documents\proto\bazed\target\debug\build\bazed-tauri-3c627ece64243281\build-script-build` (exit code: 101)
  --- stdout
  cargo:rerun-if-changed=C:\Users\Cpli\Documents\proto\bazed\crates\bazed-tauri/build.rs
  cargo:rerun-if-changed=C:\Users\Cpli\Documents\proto\bazed\crates\bazed-tauri/svelte/src
  cargo:rerun-if-changed=C:\Users\Cpli\Documents\proto\bazed\crates\bazed-tauri/svelte/package.json
  cargo:rerun-if-changed=C:\Users\Cpli\Documents\proto\bazed\crates\bazed-tauri/svelte/svelte.config.js

  --- stderr
  thread 'main' panicked at 'npm ci failed: Error { kind: NotFound, message: "program not found" }', crates\bazed-tauri\build.rs:13:10
```

**Review by gwimm — CHANGES_REQUESTED** [review]
see https://github.com/bazed-editor/bazed/pull/99#issuecomment-1374510139

**Pull request merged**